components/dashboard/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+
+type Props = {
+    percent: number;
+};
+export const ProgressBar: FC<Props> = ({ percent }) => {
+    return (
+        <div className="w-full rounded-full h-2.5 bg-gray-300 dark:bg-gray-300">
+            <div
+                className="transition-width ease-linear duration-1000 bg-blue-500 h-2.5 rounded-full"
+                style={{ width: `${percent}%` }}
+            />
+        </div>
+    );
+};
```

Comment:
**issue**: The second line with the progress bar and the details exported is creating a small visual imbalance which could be fixed with the changes described in https://github.com/gitpod-io/gitpod/pull/17791#discussion_r1212785134.

Reply:
I think I follow what's in that linked comment and will take a look. I'm not sure I follow where the progress bar is part of the imbalance you described.

Reply:
@selfcontained Let me attempt to visualize what I'm suggesting here and with the comment in https://github.com/gitpod-io/gitpod/pull/17791#discussion_r1212785134. Please, don't let this block merging this, we can always improve this later—leaving this to you. 🏓 

| BEFORE | AFTER |
|-|-|
| <img width="690" alt="Frame 1389" src="https://github.com/gitpod-io/gitpod/assets/120486/f2cdb804-2aec-4ca2-ad40-74e184b689d3"> | <img width="690" alt="Frame 1389" src="https://github.com/gitpod-io/gitpod/assets/120486/af3da6b3-265c-42ed-81f5-bd9d98ac3c5f"> |
| <img width="690" alt="Frame 1391" src="https://github.com/gitpod-io/gitpod/assets/120486/810c40b6-c916-4ab1-b0d7-cca5bc34c8d6"> | <img width="690" alt="Export-1" src="https://github.com/gitpod-io/gitpod/assets/120486/5174d5ca-3593-4201-bb06-049ebe0bf0dc"> |

Reply:
Also, not that the progress bar background also could be improved on the light theme, see also https://github.com/gitpod-io/gitpod/issues/14049. Cc @selfcontained

Reply:
Gotcha. I'll make those updates. The only part that's a bit tricky that I'm going to defer on for now is getting the progress bar to span fully underneath the close icon. I'd need to revisit the way the `<Toast/>` component is laid out. I think as we get a few more of these use cases that use toasts, it should help inform a few opinionated versions of the toast. But I'll hold off on that for now.

Reply:
DEAL—added this as point [8] in UX-38 which includes other toast component improvements. 🏀